.github/workflows/build.yml

```diff
@@ -153,7 +153,7 @@ jobs:
       run: |
         ${{env.CCACHE_SETTINGS}}
         ${{env.BUILD_DEFAULT_LINUX}}
-        cmake --build build --target test
+        ls
```

Comment:
Temporary

Reply:
Don’t think it will work as it still compile with default aes support, for pi you need disable that

Reply:
I know. I pushed a pristine version to fill the caches and to be able to reproduce the "illegal instruction". So far succeeded. See the latest commit.